sdk_container/src/third_party/coreos-overlay/coreos-base/coreos-sb-keys/coreos-sb-keys-0.0.3.ebuild

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2015 CoreOS Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Flatcar Secure Boot keys"
+HOMEPAGE=""
+SRC_URI=""
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 arm arm64 x86"
```

Please drop the unnecessary arches.

```suggestion
KEYWORDS="amd64 arm64"
```